src/azure-cli/azure/cli/command_modules/aro/_help.py

```diff
@@ -7,66 +7,66 @@
 
 
 helps['aro'] = """
-    type: group
-    short-summary: Manage Azure Red Hat OpenShift clusters.
+  type: group
+  short-summary: Manage Azure Red Hat OpenShift clusters.
 """
 
 helps['aro create'] = """
-    type: command
-    short-summary: Create a cluster.
-    examples:
-      - name: Create an OpenShift cluster
-        text: az aro create --resource-group MyResourceGroup --name MyCluster  --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet
-      - name: Create an OpenShift cluster with 5 compute nodes and Red Hat Pull Secret
-        text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet --worker-count 5 --pull-secret @pullsecret.txt
-      - name: Create a Private OpenShift cluster
-        text: az aro create --resource-group MyResourceGroup --name MyCluster --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet --apiserver-visibility Private --ingress-visibility Private
+  type: command
+  short-summary: Create a cluster.
+  examples:
+  - name: Create an OpenShift cluster
+    text: az aro create --resource-group MyResourceGroup --name MyCluster  --vnet MyVnet --master-subnet MyMasterSubnet --worker-subnet MyWorkerSubnet
+  - name: Create an OpenShift cluster with 5 compute nodes and Red Hat Pull Secret
```

Comment:
@kwoodson there are still diffs between this and the ARO-RP version.  The ARO-RP version is the one that is correct.  Please can you update this PR accordingly?